### PR TITLE
Bug/photo preview alignment 171

### LINKED ISSUE #171: Fix Photo Preview Alignment on AddNoteScreen
Description
Users are experiencing an alignment issue with the photo preview in the AddNoteScreen.tsx file. The photos in the preview section are overlapping. This issue needs to be resolved for better user experience.

Tasks

- [x] Look at the code in AddNoteScreen.tsx and find out why the photos are not aligned properly.
- [x] Make Changes to Fix It.
- [x] Make sure the fix works on small and large screens in a same way.
- [x] write test cases
- [x] Run or Simulate the app on your local machine or physical device.

Acceptance Criteria

- [x] Photos in the preview section should align properly.
    - [x] space between the preview images should match the space between upload icon and the first image in the preview.
- [x] All test cases should pass.
- [ ] App should work on your local device/machine.

Reference

Screenshot_20240822-170229~2

### PR DESCRIPTION
**Pull request by AmarHadzic**
Fix #171 

**What was changed**
Resolves and fixes the alignment issue of photos in the preview section. The photos now line up cleanly and have sufficient space in between each photo, there are no overlapping photos anymore.

**Why was it changed**
Currently, the photos in the preview section overlap each other and feature no horizontal space in between them, leading to a poor user experience. This change was necessary to ensure the photos do not overlap each other and are displayed in an organized manner, leading to a better user experience.

**How was it changed**
Modified the ```styles.mediaContainer``` to use ```flexDirection: 'row' ```, added ```flexWrap: 'wrap'```, added ```margin```, which allow the photos to align with sufficient horizontal space in between them. 

**UI after change**
Simulator Screenshot - iPhone 15 Pro Max - 2024-10-06 at 14 06 22

### CODE REVIEW

File: lib/components/photoScroller.tsx [hunk 1]
```diff
@@ -420,3 +378,31 @@ const PhotoScroller = forwardRef(
...
+    justifyContent: 'space-between', // Evenly distributes items
+    marginBottom: 10, // Adds spacing at the bottom
+  },
+  mediaItem: {
+    width: 100, // You can adjust these values
+    height: 100,
+    marginRight: 5, // Spacing between images
+  },
+});
+
```
Comment (teamomiamigo) [inline lib/components/photoScroller.tsx:408]: I see how the code is changed to help with preview alignment and any image that are rendered and placed in. I see the flex direction is there, as well as flexwrap, justify-content, and adding spacing to the bottom for anything.

**Review by teamomiamigo — COMMENTED** [review]
Overall, the changes look really good! Making the alignment of photos better through margins and the flexwrap/flexdirection/justifycontent makes it easier for new images to be rendered in and still work to different screen sizes and images as well. Great work!

**Comment by kate-holdener** [comment]
> Fix #171
> 
> **What was changed** Resolves and fixes the alignment issue of photos in the preview section. The photos now line up cleanly and have sufficient space in between each photo, there are no overlapping photos anymore.
> 
> **Why was it changed** Currently, the photos in the preview section overlap each other and feature no horizontal space in between them, leading to a poor user experience. This change was necessary to ensure the photos do not overlap each other and are displayed in an organized manner, leading to a better user experience.
> 
> **How was it changed** Modified the `styles.mediaContainer` to use `flexDirection: 'row' `, added `flexWrap: 'wrap'`, added `margin`, which allow the photos to align with sufficient horizontal space in between them.

Please include screenshots of the UI after the fix.

**Comment by rcAsironman** [comment]
@AmarHadzic You did a great job. I reviewed your work and noticed that the preview section needs a top margin. Additionally, the thumbnail for the video in the preview section is not being displayed. I will attach the previous screenshots of the thumbnail and yours for comparison.

Previous Thumbnail Screenshot
IMG_0128
IMG_0129


Yours 

IMG_0127
IMG_0130

**Comment by AmarHadzic** [comment]
@rcAsironman , those fixes were implemented and just pushed

**Comment by AmarHadzic** [comment]
Here is the UI after the changes
Simulator Screenshot - iPhone 15 Pro Max - 2024-10-06 at 14 04 55

**Review by rcAsironman — APPROVED** [review]
lgtm;

**Review by yashb196 — APPROVED** [review]
looks good 

good job Amar

**Pull request merged**